### ISSUE TITLE
Update for Ubuntu 16.04.4

### ISSUE THREAD

**Issue by JBenPiel** [chef/bento#1054]
Please update Ubuntu to the latest minor release.

**Comment by tersmitten** [comment]
Can you try to make a PR?

Something we changed recently

```
     "disk_size": "40960",
     "headless": "",
     "hostname": "wp-x.oefenweb.dev",
-    "iso_checksum": "a06cd926f5855d4f21fb4bc9978a35312f815fbda0d0ef7fdc846861f4fc4600",
+    "iso_checksum": "0a03608988cfd2e50567990dc8be96fb3c501e198e2e6efcb846d89efc7b89f2",
     "iso_checksum_type": "sha256",
-    "iso_name": "ubuntu-16.04.3-server-amd64.iso",
+    "iso_name": "ubuntu-16.04.4-server-amd64.iso",
     "memory": "2048",
     "mirror": "http://releases.ubuntu.com/releases",
-    "mirror_directory": "16.04.3",
+    "mirror_directory": "16.04.4",
     "name": "wp-amd64",
     "organization": "oefenweb",
     "preseed_path": "preseed.cfg",
```

**Comment by JBenPiel** [comment]
Resolved with PR #1056.

Thank you!
